history/subscriber/persister.py

```diff
@@ -40,6 +40,7 @@ def create_indexes(self, collection_name):
         :param collection_name: collection to create index
         """
         self.db[collection_name].create_index([('ts', pymongo.DESCENDING)])
```

Comment:
The index `self.db[collection_name].create_index([('ts', pymongo.DESCENDING)])` should be remove?